src/analysisd/analysisd.c

```diff
@@ -2394,3 +2546,156 @@ void w_init_queues(){
     /* Initialize upgrade module message queue */
     upgrade_module_input = queue_init(getDefine_Int("analysisd", "upgrade_queue_size", 128, 2000000));
 }
+
+static void load_limits(void) {
+
+    cJSON * analysisd_limits = NULL;
+    int err = load_limits_file("wazuh-analysisd", &analysisd_limits);
+
+    if (err == LIMITS_SUCCESS && analysisd_limits) {
+
+        unsigned int old_timeframe = limits.timeframe;
+        cJSON *timeframe_eps;
+
+        w_mutex_lock(&limit_eps_mutex);
+
+        if ((timeframe_eps = cJSON_GetObjectItem(analysisd_limits, "timeframe_eps"), timeframe_eps) && cJSON_IsNumber(timeframe_eps)) {
+
+            limits.timeframe = (timeframe_eps->valueint > EPS_LIMITS_MAX_TIMEFRAME ? EPS_LIMITS_MAX_TIMEFRAME :
+                (timeframe_eps->valueint < EPS_LIMITS_MIN_TIMEFRAME ? EPS_LIMITS_MIN_TIMEFRAME : timeframe_eps->valueint));
+            if (limits.timeframe != (unsigned int)timeframe_eps->valueint) {
+                mwarn("timeframe limit exceeded, value set: %d", limits.timeframe);
+            }
+        } else {
+            limits.timeframe = EPS_LIMITS_DEAULT_TIMEFRAME;
+            mwarn("timeframe doesn't found, dafault value set: %d", limits.timeframe);
+        }
+
+        cJSON *eps;
```

Comment:
Move this to line https://github.com/wazuh/wazuh/pull/13398/files#diff-54a0031895d30cd0b1e2b1589ca78822351f02ac6d1d5fe5f3e63726319df685R2559.

Reply:
Done